src/routes/transactions/mappers/multisig-transactions/multisig-transaction-execution-info.mapper.spec.ts

```diff
@@ -14,9 +14,57 @@ describe('Multisig Transaction execution info mapper (Unit)', () => {
     mapper = new MultisigTransactionExecutionInfoMapper();
   });
 
+  it('should return a MultiSigExecutionInfo with no proposer', () => {
+    const safe = safeBuilder().build();
+    const transaction = multisigTransactionBuilder()
+      .with('proposer', null)
+      .build();
+    const executionInfo = mapper.mapExecutionInfo(
```

Comment:
nitpick: new line between setup and action.

Reply:
Added in 86eb1aa66e3a2588fe4a41f2fda0f8fb03472ff8.